hocus.Dockerfile

```diff
@@ -1,52 +1 @@
-RUN echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf
-RUN curl -fsSL https://tailscale.com/install.sh | sh
-# Golang for messing with overlaybd
-RUN cd ~/ \
-    && mkdir ./go-dl \
-    && wget https://go.dev/dl/go1.20.4.linux-amd64.tar.gz -O ./go-dl/go1.20.4.linux-amd64.tar.gz \
-    && sudo tar -xvf ./go-dl/go1.20.4.linux-amd64.tar.gz -C /usr/local \
-    && rm -r ./go-dl \
-    && fish -c "set -U fish_user_paths \$fish_user_paths /usr/local/go/bin/" && echo 'export PATH="/usr/local/go/bin/:$PATH"' >> ~/.bashrc
-# Install overlaybd
-COPY --from=obd-builder /opt/overlaybd /opt/overlaybd
-COPY --from=obd-builder /etc/overlaybd /etc/overlaybd
-RUN sudo chmod u+s /opt/overlaybd/bin/overlaybd-apply
-# Install the overlaybd conversion tool
-COPY --from=obd-convertor-builder /convertor /opt/overlaybd/bin/convertor
-# Buildkite CLI + agent for running the CI jobs locally ;)
-RUN sudo wget https://github.com/buildkite/cli/releases/download/v2.0.0/cli-linux-amd64 -O /usr/bin/bk && sudo chmod +x /usr/bin/bk
-RUN curl -sL https://raw.githubusercontent.com/buildkite/agent/main/install.sh | bash && fish -c "set -U fish_user_paths \$fish_user_paths ~/.buildkite-agent/bin" && echo 'export PATH="~/.buildkite-agent/bin:$PATH"' >> ~/.bashrc
-# Hadolint :)
-RUN sudo wget https://github.com/hadolint/hadolint/releases/download/v2.12.0/hadolint-Linux-x86_64 -O /usr/bin/hadolint && sudo chmod +x /usr/bin/hadolint
-# Absurdly fast rust typo finder :3
-RUN cd ~/ && mkdir ./typos-dl && wget https://github.com/crate-ci/typos/releases/download/v1.14.5/typos-v1.14.5-x86_64-unknown-linux-musl.tar.gz -O ./typos-dl/typos.tar.gz && tar -xvf ./typos-dl/typos.tar.gz -C ./typos-dl && sudo cp ./typos-dl/typos /usr/bin/typos && rm -r ./typos-dl
-# Vale prose checker
-RUN cd ~/ && mkdir ./vale-dl && wget https://github.com/errata-ai/vale/releases/download/v2.24.1/vale_2.24.1_Linux_64-bit.tar.gz -O ./vale-dl/vale.tar.gz && tar -xvf ./vale-dl/vale.tar.gz -C ./vale-dl && sudo cp ./vale-dl/vale /usr/bin/vale && rm -r ./vale-dl
-# Dprint
-RUN curl -fsSL https://dprint.dev/install.sh | sudo bash && fish -c "set -U fish_user_paths \$fish_user_paths ~/.dprint/bin" && echo 'export PATH="~/.dprint/bin:$PATH"' >> ~/.bashrc
-# YQ
-RUN cd ~/ && mkdir ./yq-dl && wget https://github.com/mikefarah/yq/releases/download/v4.33.3/yq_linux_amd64.tar.gz -O ./yq-dl/yq.tar.gz && tar -xvf ./yq-dl/yq.tar.gz -C ./yq-dl && sudo cp ./yq-dl/yq_linux_amd64 /usr/bin/yq && rm -r ./yq-dl
-# Crane
-RUN VERSION=$(curl -s "https://api.github.com/repos/google/go-containerregistry/releases/latest" | jq -r '.tag_name') OSARCH=Linux_x86_64 bash -c 'wget -O - "https://github.com/google/go-containerregistry/releases/download/${VERSION}/go-containerregistry_${OSARCH}.tar.gz" | sudo tar -zxvf - -C /usr/local/bin/ crane'
+FROM quay.io/gorbak25/hocus-prebuilds-hocus-workspace@sha256:794bc2b876b2f3101ea69a2d1916bf5ca0866698b67a5dabf501adfc37b798c7
```

Comment:
Please add a comment where the definition of the underlying Dockerfile is and how to push an update.